docs/contribute-module.md

```diff
@@ -1,81 +1,73 @@
 
-?> _TODO_ Create tasks and conceptual information for this process
+## Overall process to contribute
 
-## Overall process :id=dev-process
+?> _TODO_ Update these tasks to make sure they're in sync with the full process (including validation). Also, create tasks and conceptual information for this process
 
-1.  Submit issue with your idea for a module. (needs issue template in [terraform-ibm-issue-tracker](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/issues/new/choose){: external}).
-1.  Meet the development prereqs.
-1.  Create repo from template (need to update [terraform-ibm-module-template](https://github.com/terraform-ibm-modules/terraform-ibm-module-template){: external}).
-1.  Add dependencies to new repo.
-1.  Update Terraform logic.
-1.  Create and run end-to-end example.
-1.  Create and run tests against example.
-1.  (Optional) Identify NIST controls
-1.  Update readme file.
-1.  Create PR.
-1.  (Optional) Onboard to IBM Cloud catalog.
+Follow these steps to contribute a new module. For more information, see the [guidelines for contributing](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/blob/main/README.md).
 
-?> _TODO_ Sync the steps with various sources
+1.  Submit an issue in the [terraform-ibm-issue-tracker](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/issues/new/choose) with your idea for a module.
 
-1. Fork the project, clone your fork, and configure the remotes:
+    ?> *TODO* (needs issue template in [terraform-ibm-issue-tracker](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/issues/new/choose)
 
-    ```bash
-    # Clone your fork of the repo into the current directory
-    git clone https://github.com/<your-username>/<repo-name>
-
-    # Navigate to the newly cloned directory
-    cd <repo-name>
-
-    # Assign the original repo to a remote called "upstream"
-    git remote add upstream https://github.com/<upstream-owner>/<repo-name>
-    ```
-
-2. If you cloned a while ago, get the latest changes from upstream:
-
-   ```bash
-   git checkout <dev_branch>
-   git pull upstream <dev_branch>
-   ```
-
-3. Create a new topic branch (off the main project development branch) to contain your feature, change, or fix:
+1.  Meet the [development prerequisites](/docs/local-dev-setup.md).
 
-   ```bash
-   git checkout -b <topic-branch-name>
-   ```
+1.  Start from an existing repo or the template:
+    ?> *TODO* Update these steps to cover everything for new and updated repos.
+    1.  If you're creating a module, follow these steps:
+        1.  Create repo from the [terraform-ibm-module-template](https://github.com/terraform-ibm-modules/terraform-ibm-module-template).
+    1.  If you're updating a module, follow these steps:
+        1.  Fork the project, clone your fork, and configure the remotes.
+1.  Clone and configure your new repo:
```

Comment:
The below is the 'normal' flow for working with any git-based project. Perhaps we can just mention it so that developers experienced with the normal flow can just skip reading each step.

Reply:
Could we just link to https://github.com/terraform-ibm-modules/documentation/blob/50e3199df08c0d226b71632256261c5ac34ec24b/docs/local-dev-setup.md#clone-and-initialize-the-repo-iddev-setup-clone

Reply:
There is more work to do on this file vs local development setup. I'm not sure we need this file. But will leave it for now.